main.py

```diff
@@ -327,9 +335,39 @@ def setup_game(self):
         pyxel.stop()
         pyxel.playm(0, loop=True)
 
+    def restart_game(self):
+        """Resets everything and restarts the game"""
+        global scroll_x
+        self.winner = False
+        self.playing = False
+        self.player = None
+        self.paused = False
+        pyxel.cls(0)
+        scroll_x = 0
+        self.setup_game()
+
+    def pause_game(self):
+        if self.paused: return
+        self.paused = True
+        self.player.active = False
+
+    def resume_game(self):
+        if not self.paused: return
```

Comment:
Hmm... Seems like flake8 doesn't like things like this (`E701 multiple statements on one line (colon)`).

```suggestion
        if not self.paused:
            return
```